hedera-node/src/main/java/com/hedera/services/state/forensics/HashLogger.java

```diff
@@ -0,0 +1,59 @@
+package com.hedera.services.state.forensics;
+
+/*-
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hedera.services.ServicesState;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class HashLogger {
+	private static final Logger log = LogManager.getLogger(HashLogger.class);
+
+	public void logHashesFor(ServicesState state) {
```

Comment:
Could this be static? If so, you can make the entire class not-intanciatable by 

1) adding the `final` keyword, `public final class HashLogger`
2) adding a private zero-arg constructor, constructor can be a no-op

Reply:
It was slightly easier to unit test this way haha. 😁